src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> readDataFromFile(String path) {
+        try {
+            return Files.lines(Path.of(path)).skip(1).toList();
```

Comment:
```suggestion
            return Files.lines(Path.of(path)).skip(OFFSET).toList();
```
let's make it more visible and easier to change if the format changes